From fb216d1a7cc981a50e4cae9179a88406663dda4e Mon Sep 17 00:00:00 2001
From: Will Woods <wwoods@redhat.com>
Date: Mon, 15 Aug 2011 11:10:59 -0400
Subject: [PATCH] fix live crash with livenet installed

parse-livenet.sh shouldn't mess with $root unless it finds a valid URL.
---
 modules.d/90livenet/parse-livenet.sh |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/modules.d/90livenet/parse-livenet.sh b/modules.d/90livenet/parse-livenet.sh
index 78fc906..323fd4a 100755
--- a/modules.d/90livenet/parse-livenet.sh
+++ b/modules.d/90livenet/parse-livenet.sh
@@ -12,8 +12,8 @@ liveurl="${liveurl#live:}"
 case "$liveurl" in
     http://*|https://*|ftp://*)
         netroot="livenet:$liveurl"
+        root="livenet" # quiet complaints from init
         rootok=1 ;;
 esac
 
-root="livenet" # quiet complaints from init
 echo '[ -e /dev/root ]' > $hookdir/initqueue/finished/livenet.sh
